client/dashboard/sites/logs/index.tsx

```diff
@@ -52,19 +61,231 @@ export function SiteLogsCallout( {
 
 function SiteLogs() {
 	const { siteSlug } = siteRoute.useParams();
+	const router = useRouter();
+	const { pathname } = router.state.location;
+
 	const { data: site } = useQuery( siteBySlugQuery( siteSlug ) );
```

Comment:
Lets use suspense, since we know the site data will have been loaded by the router

```suggestion
	const { data: site } = useSuspenseQuery( siteBySlugQuery( siteSlug ) );
```

Reply:
Thanks for the suggestion! Added in 67136512772710f909c09af093465c1f1f359978